modules/manually-create-iam-ibm-cloud.adoc

```diff
@@ -2,10 +2,21 @@
 //
 // * installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.adoc
 // * installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.adoc
+// * installing/installing_ibm_power/installing-ibm-power-vs-customizations.adoc
+
+ifeval::["{context}" == "installing-ibm-power-vs-customizations"]
+:ibm-power-vs:
+endif::[]
 
 :_content-type: PROCEDURE
+ifndef::ibm-power-vs[]
```

Comment:
Similar to a previous comment, I suggest creating an `ifeval` for the other two IBM Cloud assemblies that this module is used in. For example, you could use `ibm-vpc` as an attribute for both of those assemblies. Happy to discuss.

Reply:
After discussing this, please create new ifeval statements for the IBM Cloud VPC assemblies with the ibm-vpc attribute.